src/promptflow/promptflow/_sdk/schemas/_flow.py

```diff
@@ -58,6 +60,25 @@ class EagerFlowSchema(BaseFlowSchema):
     """Schema for eager flow."""
 
     # path to flow entry file.
-    path = LocalPathField(required=True)
+    path = LocalPathField(required=False)
```

Comment:
we should remove path follow this PR? https://github.com/microsoft/promptflow/pull/1861/files#diff-a40f32e9c2dff3899f142d10888a04f41f93d5bab55880380dfe0abf588ea847

Reply:
it will be removed after that draft PR merged, but it depends on some update from batch engine side so I can't change it directly in this PR. BTW, need to close this PR and reopen based on a dev/xxx branch